src/resampling.jl

```diff
@@ -509,13 +512,15 @@ struct.
   training and evaluation respectively.
 """
 struct PerformanceEvaluation{M,
+                             Model,
                              Measurement,
                              Operation,
                              PerFold,
                              PerObservation,
                              FittedParamsPerFold,
                              ReportPerFold} <: MLJType
     measure::M
+    model::Model
```

Comment:
Nit-picky suggestion. We frequently use `M` for a model type elsewhere. For consistency, can we do something like:

```suggestion
struct PerformanceEvaluation{Measure,
                             M,
                             Measurement,
                             Operation,
                             PerFold,
                             PerObservation,
                             FittedParamsPerFold,
                             ReportPerFold} <: MLJType
    measure::Measure
    model::M
```